samples/sampler/stories/default/TabbedPanels.js

```diff
@@ -98,7 +97,13 @@ const I18nTabbedPanels = I18nContextDecorator({rtlProp: 'rtl'}, I18nTabbedPanels
 storiesOf('Agate', module)
 	.add(
 		'TabbedPanels',
-		() => (<I18nTabbedPanels />),
+		() => (
+			<I18nTabbedPanels
+				duration={number('duration', Config, 500)}
```

Comment:
Is it simply to make the TabbedPanels knob visible before the Global knob?
What are the criteria for what's remain here and what's put in TabbedPanels?

Reply:
We use the `rtl` prop to conditionally render the left and right arrow icons based on locale. The logic for this functionality is left in `TabbedPanles` and the other knobs that don't rely on the `rtl` prop is passed to `I18nTabbedPanels`.

Reply:
Hm.. I see